### PR TITLE
Update SIG Docs leads

### PR DESCRIPTION
**Pull request by zacharysarah**
This PR updates leads for SIG Docs: specifically, that @chenopis stepped down as co-chair (we'll miss you!) and @jaredbhatti has replaced him (welcome!).

/assign @Bradamant3 @chenopis @jaredbhatti 
/sig-docs

See https://github.com/kubernetes/org/pull/659

### CODE REVIEW

File: sig-docs/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Covers documentation, doc processes, and doc publishing for Kubernetes.
 ### Chairs
 The Chairs of the SIG run operations and processes governing the SIG.
 
-* Andrew Chen (**[@chenopis](https://github.com/chenopis)**), Google
+* Jared Bhatti (**[@jaredbhatti](https://github.com/jaredbhatti)**), Google
```
Comment (Bradamant3) [inline sig-docs/README.md]: Isn't this file generated? So the only file to change is the YAML file?

File: sig-list.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
...
+|[Docs](sig-docs/README.md)|docs|* [Jared Bhatti](https://github.com/jaredbhatti), Google<br>* [Zach Corleissen](https://github.com/zacharysarah), Linux Foundation<br>* [Jennifer Rondeau](https://github.com/bradamant3), VMware<br>|* [Slack](https://kubernetes.slack.com/messages/sig-docs)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)|* Regular SIG Meeting: [Tuesdays at 17:30 UTC (weekly - except fourth Tuesday every month)](https://docs.google.com/document/d/1zg6By77SGg90EVUrhDIhopjZlSDg2jCebU-Ks9cYx0w/edit)<br>* APAC SIG Meeting: [Wednesdays at 02:00 UTC (monthly - fourth Wednesday every month)](https://docs.google.com/document/d/1zg6By77SGg90EVUrhDIhopjZlSDg2jCebU-Ks9cYx0w/edit)<br>
```
Comment (Bradamant3) [inline sig-list.md]: And isn't this one also generated?

Comment (zacharysarah) [reply sig-list.md]: Yes, they’re generated; the repo requires PRs to run `make generate` and include the results.

Comment (zacharysarah) [reply sig-list.md]: Yes, they’re generated; the repo requires PRs to run `make generate` and include the results.

**Comment by Bradamant3** [comment]
again, not sure I have rights, but here you go:

/approve

File: sigs.yaml [hunk 3]
```diff
@@ -1076,8 +1076,8 @@ sigs:
     label: docs
     leadership:
       chairs:
-      - name: Andrew Chen
-        github: chenopis
+      - name: Jared Bhatti
```
Comment (nikhita) [inline sigs.yaml]: Consider adding Andrew as an Emeritus lead? :)


Example - https://github.com/kubernetes/community/blob/d335e174085078db1a3b12ae86753c66274d1813/sigs.yaml#L325

Comment (zacharysarah) [reply sigs.yaml]: @nikhita What a wonderful idea! 💛

@Bradamant3 I updated your current employer as well, if that's OK with you. Please let me know.

Comment (Bradamant3) [reply sigs.yaml]: lol just saw this after I started what I thought would be a follow-up PR to do just this. Thank you! `<subsides>`

Comment (nikhita) [reply sigs.yaml]: I guess it should be changed here as well? :grimacing: 

https://github.com/kubernetes/community/blob/20c1e33a741f238ec0428f7fdc096ca1ff0d862d/sigs.yaml#L2644-L2646

**Pull request merged**